src/renderer/families/cosmos/DelegationFlowModal/fields/ValidatorsField.js

```diff
@@ -265,14 +263,18 @@ const ValidatorField = ({
     ],
   );
 
+  const formatMax = +formatCurrencyUnit(unit, max);
```

Comment:
there is an accidental `+` here?

Reply:
Not really in that case :sweat_smile: i need a formatted number to check if the available max is lower than 0.
The component is shared with tron so it makes for weird stuff like this :shrug:

Reply:
ok, we really shouldn't "parse the formatter" , i don't understand why we don't do somethign else like `max.times(10 ** unit.magnitude).toNumber()` ?


i don't like the `+something` pattern at all because it looks like a typo, i prefer we explicitly use parseInt if it makes sense, here we should try to not format and parse again

Reply:
I forgot to ask this, but i'm curious why we actually use floating value. we should try to stay in the "satoshis" domain of units. to me we only need to go back to "decimals" domain at the parser and formatter time, which are two extremity. forcing us to do so will be a good practice in a future consideration of a switch to `bigint` – which will not have "floating points".